packages/function-tree/README.md

```diff
@@ -464,8 +464,11 @@ execute.on('error', (error, execution, payload) => {})
 // When a function tree is executed
 execute.on('start', (execution, payload) => {})
 
-// When a function tree ends its execution
-execute.on('end', (execution, payload) => {})
```

Comment:
@christianalfoni Why is the end event removed from the docs? 
Is it no public API anymore?